Xabe.FFmpeg/Conversion/IConversion.cs

```diff
@@ -54,11 +54,26 @@ public interface IConversion
         IConversion ExtractNthFrame(int frameNo, Func<string, string> buildOutputFileName);
 
         /// <summary>
-        /// Builds the -framerate option for this conversion
+        /// Builds a video from a directory containing one or more sequentially named images
         /// </summary>
-        /// <param name="frameRate"></param>
+        /// <param name="startNumber">The number of the image to start building video from</param>
+        /// /// <param name="buildInputFileName"> Delegate Function to build up custom filename when inputting multiple files </param>
         /// <returns>IConversion object</returns>
-        IConversion SetFrameRate(double frameRate);
+        IConversion BuildVideoFromImages(int startNumber, Func<string, string> buildInputFileName);
+
+        /// <summary>
+        /// Builds the -framerate option for the output of this conversion
+        /// </summary>
+        /// <param name="frameRate">The desired framerate of the output</param>
+        /// <returns>IConversion object</returns>
+        IConversion SetOutputFrameRate(double frameRate);

```

Comment:
We cannot rename methods in IConversion. It's already used and will break the contract. This require increate the major version of packages.

Reply:
I've renamed it back to SetFramerate